app/javascript/article-form/components/Title.jsx

```diff
@@ -1,5 +1,14 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
+// We use this magic Textarea component for title field because it's automatically
+// resizable. Even though it looks like a classic input, if you enter long title
+// it would wrap the text to the next line automatically resizing itself. It helps keeping
+// the entire layout the way it is without having unnecessary scrolling and white spaces.
+// Keep in mind this is what happens only here - in preact component.
+// I'm mentioning this because the entire "Write a post" view is a preact component
+// BUT it is also a classic .html.erb view which is being loaded BEFORE this component
+// to give a feeling of blazing fast page load. And we do NOT use this magic autoresizing
+// functionality on .html.erb view because there's no point of it.
```

Comment:
We could probably cut this part of the comment, IMO.

Reply:
Even though it's not 100% related to that specific `title` field, I do think it provides some additional context which can help understand behavior of that field in certain edge cases (for example when a long title is entered) so I'm leaning towards keeping it as it is.